### PR TITLE
[c2][fuzzer] Add fuzzer support

### PR DESCRIPTION
**Pull request by zhangyichix**
* Add fuzzer support

Tracked-On: OAM-112331

### CODE REVIEW

**Comment by zhangyichix** [comment]
@TianmiChen @Shao-Feng @YuanjunHuang Please help to review, thanks!

File: c2_utils/src/mfx_va_allocator.cpp [hunk 1]
```diff
@@ -595,43 +553,6 @@ mfxStatus MfxVaFrameAllocator::CreateSurfaceFromGralloc(const IMfxGrallocModule:
...
-    VADRMPRIMESurfaceDescriptor desc;
-    MFX_ZERO_MEMORY(desc);
-
-    desc.fourcc = va_fourcc;
-    desc.width = width;
-    desc.height = height;
-    desc.num_objects = 1;
-    desc.objects[0].fd = info.prime;
-    desc.objects[0].size = decode_target ? info.pitches[0] * ((height + 31) & ~31) * 1.5 : info.pitches[0] * ((height + 15) & ~15) * 1.5;
-    if (HAL_PIXEL_FORMAT_NV12_Y_TILED_INTEL == info.format || HAL_PIXEL_FORMAT_P010_INTEL == info.format)
```
Comment (TianmiChen) [inline c2_utils/src/mfx_va_allocator.cpp:608]: Why do we have to remove this part?

Thanks,
Tianmi

Comment (zhangyichix) [reply c2_utils/src/mfx_va_allocator.cpp:608]: Got it wrong, my local branch is messed up, will fix it. sorry...

**Comment by zhangyichix** [comment]
updated.
Still under testing...

**Review by TianmiChen — APPROVED** [review]

**Pull request merged**